packages/contracts/src/current/protocol/TokenTransferProxy/TokenTransferProxy.sol

```diff
@@ -16,10 +16,10 @@
 
 */
 
-pragma solidity ^0.4.11;
+pragma solidity ^0.4.19;
```

Comment:
Same as above. When we do start making changes to the proxy, we should also keep a `TokenTransferProxy_v1` copy in the previous directory (just occurred to me, we should have done this for the Exchange as well).